### PR TITLE
docs(LLM): LLM notebooks

### PR DESCRIPTION
**Pull request by omrihaber**

### CODE REVIEW

File: elemeta/nlp/extractors/high_level/capital_letters_ratio.py [hunk 1]
```diff
@@ -1,38 +1,52 @@
...
-        name: Optional[str]
-            name to of the metadata of not given will extract the name from the class name
-        """
         super().__init__(name)
 
     def extract(self, text: str) -> float:
-        """case ratio calculator
-        returns the ratio of capital letters / length
+        """
+        Calculates the ratio of capital letters to lower letters in the given text.
```
Comment (BigicecreamTaken) [inline elemeta/nlp/extractors/high_level/capital_letters_ratio.py:39]:
```suggestion
        Calculates the ratio of capital letters to all letters in the given text.
```

File: elemeta/nlp/extractors/high_level/emoji_count.py [hunk 2]
```diff
@@ -2,37 +2,51 @@
...
+
+    Methods
+    -------
+    extract(text)
+        Counts the number of emojis in the given text.
+
+    Examples
+    --------
+    >>> emoji_coiunter = EmojiCount()
+    >>> text = "🤔 🙈 me así, bla es se 😌 ds 💕👭👙"
```
Comment (BigicecreamTaken) [inline elemeta/nlp/extractors/high_level/emoji_count.py:30]: lets change the example

Comment (omrihaber) [reply elemeta/nlp/extractors/high_level/emoji_count.py]: changed

File: elemeta/nlp/extractors/low_level/hinted_profanity_token_count.py [hunk 3]
```diff
@@ -3,14 +3,21 @@
...
     Implementation of AbstractTextMetafeatureExtractor class that count the number profanity words
+
+    Example
+    -------
+    >>> from elemeta.nlp.extractors.low_level.hinted_profanity_token_count import HintedProfanityTokensCount
+    >>> from nltk import word_tokenize
+    >>> text = "Once I was afraid, I was petrified"
+    >>> profanity_token_counter = HintedProfanityTokensCount(word_tokenize)
+    >>> result = profanity_token_counter(text)
+    >>> print(result)  # Output: 0
```
Comment (BigicecreamTaken) [inline elemeta/nlp/extractors/low_level/hinted_profanity_token_count.py:20]: Let's give an example with Profanity

**Review by BigicecreamTaken — APPROVED** [review]
Good Job Omri!
Thank you for the contribution.

**Pull request merged**